pyswmm/swmm5.py

```diff
@@ -501,8 +504,8 @@ def getSimUnit(self, unittype):
         >>> swmm_model.swmm_close()
         """
         value = ctypes.c_int()
-        errcode = self.SWMMlibobj.swmm_getSimulationUnit(unittype,
-                                                         ctypes.byref(value))
+        errcode = self.SWMMlibobj.swmm_getSimulationUnit(
```

Comment:
I would "un" line break these and get ciocheck running.  These should all be solved with that